### ISSUE TITLE
Windows app not working

### ISSUE THREAD

**Issue by lordmen99** [AgoraIO/Live-Shop-Use-Case#2]
After build and run exe file, cant load. please fix it

**Comment by lordmen99** [comment]
can working, but not have sound in viewer android app

**Comment by brianxcli** [comment]
@lordmen99  Can you give more information, I cannot help you without any clue.

**Comment by plutoless** [comment]
@lordmen99  pls check your phone volume settings. if everything is correct but the sound is still not working, kindly provide your channel name and appid so that we can take a look.

**Comment by lordmen99** [comment]
i still use default app id from demo app. for volume my phone have increased maximum. Btw, what for this app? i mean, user can get product with answer question or just buy it from broadcaster?

**Comment by plutoless** [comment]
@lordmen99 are you using a xiaomi note 5a phone? I took a quick look at our media backend and see the audio packets have been properly received from Android side while the audio playback frequency is also correct. If it's still not working, try switching another device to see if that helps.

There are many variants that you can play around with liveshop, in this one we demo here you can view video broadcasts of products and buy directly from your app, at the same time you will receive quiz and you will get buying bonus if you get it right

**Comment by lordmen99** [comment]
yes correct. maybe any demo running in playstore with same concept?

**Comment by plutoless** [comment]
@lordmen99 no for the moment we don't have official release app to demo this. Also this demo app only shows you the major steps how you can integrate with Agora SDK to achieve this kind of app. The demo itself does not include any real world business logics.
